numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -928,25 +928,24 @@ def add_newdoc(place, name, doc):
 
     The Heaviside step function is defined as::
 
-                             0   if x < 0
-        heaviside(x, h0) =  h0   if x == 0
-                             1   if x > 0
```

Comment:
As far as I can tell only `heaviside` does this, the only other irregularity being `y1`/`y2` in [the docs of `modf`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.modf.html) (every other instance calls these `out1`/`out2`.

I believe I found the source of the double `$PARAMS$`: [`isnat` has none](https://github.com/numpy/numpy/blob/master/numpy/core/code_generators/ufunc_docstrings.py#L1723-L1725). Is this minor enough, or should I open a new issue with this?